cumulus/polkadot-parachain/src/command.rs

```diff
@@ -164,6 +170,32 @@ fn load_spec(id: &str) -> std::result::Result<Box<dyn ChainSpec>, String> {
 				&include_bytes!("../chain-specs/asset-hub-kusama.json")[..],
 			)?),
 
+		// -- Asset Hub Rococo
+		"asset-hub-rococo-dev" =>
+			Box::new(chain_spec::asset_hubs::asset_hub_rococo_development_config()),
+		"asset-hub-rococo-local" =>
+			Box::new(chain_spec::asset_hubs::asset_hub_rococo_local_config()),
+		// the chain spec as used for generating the upgrade genesis values
+		"asset-hub-rococo-genesis" => Box::new(chain_spec::asset_hubs::asset_hub_rococo_config()),
+		// the shell-based chain spec as used for syncing
+		"asset-hub-rococo" =>
+			Box::new(chain_spec::asset_hubs::AssetHubRococoChainSpec::from_json_bytes(
+				&include_bytes!("../chain-specs/asset-hub-rococo.json")[..],
+			)?),
+
+		// -- Asset Hub Wococo
+		"asset-hub-wococo-dev" =>
+			Box::new(chain_spec::asset_hubs::asset_hub_wococo_development_config()),
+		"asset-hub-wococo-local" =>
+			Box::new(chain_spec::asset_hubs::asset_hub_wococo_local_config()),
+		// the chain spec as used for generating the upgrade genesis values
+		"asset-hub-wococo-genesis" => Box::new(chain_spec::asset_hubs::asset_hub_wococo_config()),
+		// the shell-based chain spec as used for syncing
```

Comment:
same

Reply:
@georgepisaltu thanks,
so removing this comment, because AssetHubWococo is deployed from the scratch with custom runtime and not shell runtime